torchaudio/functional/functional.py

```diff
@@ -1666,7 +1698,24 @@ def pitch_shift(
     waveform_stretch = torch.istft(
         spec_stretch, n_fft=n_fft, hop_length=hop_length, win_length=win_length, window=window, length=len_stretch
     )
-    waveform_shift = resample(waveform_stretch, int(sample_rate / rate), sample_rate)
+    return waveform_stretch
+
+
+def _pitch_shift_postprocess(
+    waveform_shift: Tensor,
+    shape: List[int],
+) -> Tensor:
+    """
+    PitchShift helper function to process after resampling step.
+
+    Args:
+        waveform_shift (Tensor): The input shifted waveform of shape `(..., time)`.
+        shape (List[int]): shape of initial waveform
+
+    Returns:
+        Tensor: The pitch-shifted audio waveform of shape `(..., time)`.
+    """
```

Comment:
The description needs to be changed. It returns the waveform with `shape` dimensions.